eth/api_tracer.go

```diff
@@ -701,7 +701,7 @@ func (api *PrivateDebugAPI) computeTxEnv(blockHash common.Hash, txIndex int, ree
 
 		if tokenFeeUsed {
 			fee := new(big.Int).SetUint64(gas)
-			if block.Header().Number.Cmp(common.TIPTRC21Fee) > 0 {
+			if api.config.IsTIPTRC21Fee(block.Header().Number) {
```

Comment:
Many places changing from block number to ChainConfig.IsXXX still need to reviewed and be fixed.